data/presets/amenity/cafe/teahouse.json

```
@@ -0,0 +1,30 @@
+{
+    "icon": "maki-cafe",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "cafe",
+        "cuisine": "tea"
+    },
+    "reference": {
+        "key": "cuisine",
+        "value": "tea"
+    },
```

```suggestion
    "reference": {
        "key": "amenity",
        "value": "cafe"
    },
```

I am not sure about the `tea` `reference` here. If we keep the tea, we should add a data item that is about tee. It looks like there is not data item, yet, but a Wiki page https://wiki.openstreetmap.org/wiki/Tag:cuisine%3Dtea
The docs here have some details on how to add a data item, search for "info-i".

However, the data item for tea should still say something similar to the wiki page "A drink made by steeping tea leaves in hot water." which does not describe the feature.

The clearest info-i is IMO the amenity=cafe.